execution_engine/src/storage/trie_store/db.rs

```diff
@@ -2,76 +2,48 @@
 
 use std::{
     collections::HashMap,
+    path::{Path, PathBuf},
     sync::{Arc, Mutex},
 };
 
-use casper_types::{bytesrepr, Key, StoredValue};
-use lmdb::{Database, DatabaseFlags};
+use rocksdb::{
```

Comment:
There's a comment in line 1 that mentions LMDB.

Reply:
Fixed in 213bf27a50f00424def44d1557567f3f3d184e21